cc-pk-service/src/routes/pkServiceRoutes.ts

```diff
@@ -0,0 +1,47 @@
+// PK service route module
+import { Router, Request, Response } from "express";
+import {getReusables, getSurveys, getItemInfo, userId} from "../handleData";
+import { Error, ItemInfo, Survey, Item} from '../types';
+
+// Express router for accessing the defined paths
+const pkRouter = Router();
+
+// Route which returns "pong"
+// http://localhost:5123/v1/system/ping
+pkRouter.get('/system/ping', (_:Request, res:Response) => {
+  res.send('pong');
+});
+
+// Route which returns user's items
+// http://localhost:5123/v1/users/:usedId/items
+pkRouter.get('/users/:userId/items', async (req:Request, res:Response) => {

```

Comment:
is it possible to move the try-catch -blocks from handleData to this level?

Reply:
I tried moving stuff around, but it seems like axios doesn't like it when its requests aren't inside try-catch -blocks. This is the error that I got:

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "AxiosError: Request failed with status code 401".] {
  code: 'ERR_UNHANDLED_REJECTION'
}